SynthSeg/training_with_tfrecords.py

```diff
@@ -183,21 +228,87 @@ def load_model(
                 for (key, value) in getmembers(nrn_layers, isclass)
                 if key != "Layer"
             }
+
             custom_objects = {
                 **custom_l2i,
                 **custom_nrn,
                 "tf": tf,
                 "keras": tf.keras,
                 "loss": IdentityLoss().loss,
+                "DiceLoss": DiceLoss
```

Comment:
What happens if we want to continue training for a model with the `WeightedL2Loss`? I guess we have add either the dice or the l2 loss, or maybe both, not sure.